src/main/java/controller/BankController.java

```diff
@@ -0,0 +1,134 @@
+package controller;
+
+import bank.GeneralMember;
+import bank.BankService;
+import bank.SavingsMember;
+import interest.InterestService;
+import util.Appconfig;
+import view.InputView;
+import view.OutputView;
+
+import java.math.BigDecimal;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static util.ErrorMessage.WRONG_FORM;
+
+
+public class BankController {
+
+    private final InputView inputView;
+    private final OutputView outputView;
+
+
+    Appconfig appConfig = new Appconfig();
+    BankService bankService = appConfig.bankService();
+    InterestService interestService = appConfig.interBankService();
```

Comment:
```suggestion
    private final InterestService interestService = appConfig.interBankService();
```
접근 제어자를 알맞게 설정하고, final 키워드를 이용하여 해당 필드의 쓰임을 명시해주는 것이 중요합니다